Symbl-Powered-Agora-master/src/subComponents/ScreenshareButton.tsx

```diff
@@ -49,6 +49,8 @@ const ScreenshareButton = (props: any) => {
         style={[style.buttonIcon, { tintColor: primaryColor }]}
         resizeMode={'contain'}
       />
+      <Text 
+        style={{backgroundColor: '#fff', width: 46}}>{ screenshareActive? 'Stop' : 'Start'}</Text>
```

Comment:
Perhaps we can update this to `Screen Share` instead of Start / Stop?

Reply:
<img width="826" alt="Screenshot 2021-11-15 at 10 13 39 AM" src="https://user-images.githubusercontent.com/88789145/141723852-78483805-6aa8-46d8-a75d-e6e01a0c23a7.png">
<img width="831" alt="Screenshot 2021-11-15 at 10 13 59 AM" src="https://user-images.githubusercontent.com/88789145/141723857-16704dd2-b705-4cb2-877c-a3cd41e8ddea.png">

Hey @mjabali updated! This is how it looks now (attached screenshots)